tests/element_test_vectors.rs

```diff
@@ -616,6 +674,14 @@ mod native_element_tests {
 
         fn round_trip_skip_list() -> SkipList {
             &[
+                // These tests fail because the ion-c writer doesn't fully step out of the last
+                // container it writes in some circumstances, leading to a panic.
+                "ion-tests/iontestdata/good/testfile25.ion",
+                "ion-tests/iontestdata/good/testfile33.ion",
+                "ion-tests/iontestdata/good/non-equivs/timestamps.ion",
+                "ion-tests/iontestdata/good/equivs/timestamps.ion",
+                "ion-tests/iontestdata/good/equivs/timestampFractions.ion",
+                // -----
```

Comment:
🗺️ I plan to leave the `ion-c`-writer-related failures on the skip list for now and move on to wiring up the `NativeElementWriter`. I assume it also has bugs, but that's a higher priority for investing time in fixes.

Reply:
Yes, I like this plan.